lib/elixir/unicode/PropList.txt

```diff
@@ -0,0 +1,27 @@
+# PropList-8.0.0.txt
+# Date: 2015-05-16, 17:50:38 GMT [MD]
+#
+# Unicode Character Database
+# Copyright (c) 1991-2015 Unicode, Inc.
+# For terms of use, see http://www.unicode.org/terms_of_use.html
+# For documentation, see http://www.unicode.org/reports/tr44/
+
+# ================================================
+
+0009..000D    ; White_Space # Cc   [5] <control-0009>..<control-000D>
```

Comment:
It is fine so let's keep it as separate file still because we'll copy the section as is when there is a new unicode release.